### ISSUE TITLE
2.3b6: batou.lib.file.File: Incorrectly formatted error message when combining an explicit template and content

### ISSUE THREAD

**Issue by frlan** [flyingcircusio/batou#348]
Accidentality called a File() with both, an explicit template and content:

```python
        self +=File(
           "myfile",
           source="my_template",
           content="Hello World!")
```

and got this error: 

```
ERROR: ValueError('Only one of either "{}" or "source" are allowed.', 'content')
This /might/ be a batou bug. Please consider reporting it.
```

I think this is caused by this code from file.py
```python
        if self.source and getattr(self, self._content_source_attribute):
            raise ValueError(
                'Only one of either "{}" or "source" are allowed.',
                format(self._content_source_attribute),
            )
```

(batou 2.3b6)

**Comment by stevleibelt** [comment]
Thanks @frlan for reporting this issue. Could you please extend your report with your expected error message as an example?

Best regards,
Stev

**Comment by elikoga** [comment]
Looks like:

```
        if self.source and getattr(self, self._content_source_attribute):
            raise ValueError(
                'Only one of either "{}" or "source" are allowed.'.format(
                    self._content_source_attribute
                ),
            )
```

was intended. I'm trying to add this error to one of the tests, before I add it though

**Comment by frlan** [comment]
Why not using something like `f'Only one of either "{self._content_source_attribute}" or "source" are allowed.'`  – we don't support older Python versions anyway.

**Comment by elikoga** [comment]
closed in #371
